code/modules/spacepods/spacepod.dm

```diff
@@ -73,6 +73,8 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
+	equipment_system.installed_modules = list()
```

Comment:
Making `installed_modules` into a list should just be done [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/spacepods/equipment.dm#L47).